### DISCUSSION TITLE
macOS Ventura's createinstallmedia incompatibilities with Yosemite

### DISCUSSION THREAD

**Discussion by khronokernel** [dortania/OpenCore-Legacy-Patcher#1038] [Reports being investigated]
Process crashes when invoked, originally reported by Jazzzny

**Comment by Jazzzny** [comment]
After further research, it appears that Apple has set a new minimum target for `createinstallmedia` as 10.12 Sierra. However, all binaries are still compatible with 10.10 Yosemite and 10.11 El Capitan.
However, `OSInstallerSetup.framework` inside Contents/Frameworks is not compatible with 10.10 Yosemite.
Fortunately, replacing 13 Ventura's `OSInstallerSetup.framework` with the one bundled with the 12 Monterey installer fixes `createinstallmedia` on 10.10 Yosemite:
![Screen Shot 2023-04-02 at 7 03 34 PM](https://user-images.githubusercontent.com/75343012/229384005-65bf5351-350c-4945-81c3-258304e3e4a0.png)

- Rename [Installer app]/Contents/Frameworks/OSInstallerSetup.framework
- Copy over OSInstallerSetup.framework from 12.6's Installer (Attached) to [Installer app]/Contents/Frameworks/OSInstallerSetup.framework
[OSInstallerSetup.framework.zip](https://github.com/dortania/OpenCore-Legacy-Patcher/files/11133603/OSInstallerSetup.framework.zip)
- Verify that createinstallmedia is functional

After the installer USB is created, the modified /Volumes/Install macOS Ventura/[Installer app]/Contents/Frameworks/OSInstallerSetup.framework should be deleted and the original should be restored.
